ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CompanyMapper.java

```diff
@@ -4,47 +4,57 @@
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.cmc.ccd.domain.CCDApplicant;
 import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.CCDParty;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPartyType;
 import uk.gov.hmcts.cmc.domain.models.party.Company;
 
 @Component
 public class CompanyMapper {
 
     private final AddressMapper addressMapper;
     private final RepresentativeMapper representativeMapper;
+    private final TelephoneMapper telephoneMapper;
 
     @Autowired
-    public CompanyMapper(AddressMapper addressMapper, RepresentativeMapper representativeMapper) {
+    public CompanyMapper(AddressMapper addressMapper, RepresentativeMapper representativeMapper,
```

Comment:
since params have gone in next line, would suggest to follow our standard approach

Reply:
done